detox/src/devices/Fbsimctl.js

```diff
@@ -173,6 +173,14 @@ class Fbsimctl {
     await this._execFbsimctlCommand(options);
   }
 
+  async resetContentAndSettings(udid) {
```

Comment:
that's what @formatlos suggested, the UDID is not a part of the API on any function, but is used internally (each  `Device.js` instance has it's own deviceId)